lib/osm_community_index.rb

```diff
@@ -0,0 +1,12 @@
+module OsmCommunityIndex
+  require "yaml"
+
+  def self.community_index
+    @community_index ||= community_index_from_json
+  end
+
+  def self.community_index_from_json
+    json_file = Rails.root.join("node_modules/osm-community-index/dist/resources.json")
```

Comment:
Is this the best way to link the dataset?  Does OCI have an API we could call?

Reply:
Thinking about it a bit, that would be more complicated to sync.  Maybe OSM should be the system of record for communities.

Reply:
Thanks for taking a look @openbrian (and sorry for the delay replying 😬). My intention with using Open Community Index was to follow the "single source of truth" principle. OCI already have the data we need to render the page on osm.org, so seems rude not to try to use it and share data across the OSM ecosystem. I do appreciate that a shorter-term solution of just hardcoding the data would have been easier to achieve, and we could always fall back to that if this approach doesn't pan out.

> Maybe OSM should be the system of record for communities.

Do you mean OCI or do you have something else in mind (if so I'd love to hear what you're thinking)?